components/02-elements/button/_button.scss

```diff
@@ -44,6 +45,7 @@ $button-icon-hover-color          : $color-primary !default;
     margin: $button-margin;
     padding: $button-padding;
     border: $button-border;
+    border-radius: $button-border-radius;
     outline: $button-outline;
     background: $button-background;
     font-family: $button-font-family;
```

Comment:
1. In styles file, to &--icon modifier and `.button__icon` element, please add `margin: auto` (on Safari 10.1.2 icon is align to left and this rule solve the problem
2. To the same element please add transition - to change `fill` on hover